src/components/TodoList/TodoList.tsx

```diff
@@ -14,87 +28,59 @@ export const TodoList: React.FC = () => (
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
+      {todos && todos.map((todo) => {
+        const {
+          id, title, userId, completed,
+        } = todo;
 
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">6</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">
-            qui ullam ratione quibusdam voluptatem quia omnis
-          </p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">8</td>
-        <td className="is-vcentered">
-          <span className="icon" data-cy="iconCompleted">
-            <i className="fas fa-check" />
-          </span>
-        </td>
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-success">quo adipisci enim quam ut ab</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
+        return (
+          <tr
+            data-cy="todo"
+            className={cn({ 'has-background-info-light': selected?.id === id })}
+            key={id}
+          >
+            <td className="is-vcentered">{id}</td>
+            {completed
+              ? (
+                <td className="is-vcentered">
+                  <span className="icon" data-cy="iconCompleted">
+                    <i className="fas fa-check" />
+                  </span>
+                </td>
+              )
+              : <td className="is-vcentered" />}
+            <td className="is-vcentered is-expanded">
+              <p className={cn({
+                'has-text-danger': !completed,
+                'has-text-success': completed,
+              })}
+              >
+                {title}
+              </p>
+            </td>
+            <td className="has-text-right is-vcentered">
+              <button
+                data-cy="selectButton"
+                className="button"
+                type="button"
+                onClick={() => {
+                  setUserId(userId);
+                  selectTodo(todo);
+                }}
+              >
+                <span className="icon">
+                  <i className={cn('far', {
+                    'fa-eye': selected?.id !== id,
```

Comment:
you checked is current todo selected several times, let's create separate boolean variable inside the callback with the result of this check and will use it